packages/web3-net/test/unit/web3_net_methods.test.ts

```diff
@@ -0,0 +1,29 @@
+import { Web3Net } from '../../src';
+import * as rpcMethodWrappers from '../../src/rpc_method_wrappers';
+
+jest.mock('../../src/rpc_method_wrappers');
+
+describe('web3_eth_methods_no_parameters', () => {
+	let web3Net: Web3Net;
+
+	beforeAll(() => {
+		web3Net = new Web3Net('http://127.0.0.1:8545');
+	});
+
+	describe('should call RPC method', () => {
+		it('getId', async () => {
+			await web3Net.getId();
```

Comment:
Needs to test `returnType` implementation [e.g.](https://github.com/ChainSafe/web3.js/blob/ea41c556130558b37305767a39f25b4306dc0e50/packages/web3-eth/test/unit/web3_rpc_method_wrappers_with_parameters.test.ts#L56)